tests/testParquet.py

```diff
@@ -0,0 +1,149 @@
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function
+
+import os, shutil
+import unittest
+import tempfile
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+from pandas.util.testing import assert_frame_equal
+
+import lsst.utils.tests
+
+from lsst.qa.explorer.parquetTable import ParquetTable, MultilevelParquetTable
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+
+def setup_module(module):
+    lsst.utils.tests.init()
+
+class ParquetTableTestCase(unittest.TestCase):
+    """Test case for ParquetTable
+    """
+    testFilename = 'simple_test.parq'
```

Comment:
Should use a per-test-function-unique (at least) name to avoid problems when pytests runs multiple tests in parallel.

It looks like you may be avoiding that already by using `tempfile`, but if you can use `lsst.utils.tests.getTempFilePath`, that'd be more standard and it may provide some useful functionality (e.g. retaining the file with a meaningful name when a test fails).